docker/script/entrypoint.sh

```diff
@@ -23,6 +23,19 @@ export \
 install_requirements() {
     # Install custom python package if requirements.txt is present
     if [[ -e "$AIRFLOW_HOME/$REQUIREMENTS_FILE" ]]; then
+      if grep -- "-c " "$AIRFLOW_HOME/$REQUIREMENTS_FILE"
+      then
+          # --constraint found
+          :
+      else
+          if grep -- "--constraint " "$AIRFLOW_HOME/$REQUIREMENTS_FILE"
+          then
+              # --constraint found
+              :
+          else
+              echo "WARNING: Constraints should be specified for requirements.txt. Please see https://docs.aws.amazon.com/mwaa/latest/userguide/working-dags-dependencies.html#working-dags-dependencies-test-create"
```

Comment:
Why not negate this and use an and operator to get rid of the empty branches?

Reply:
Done